x/authz/client/cli/tx.go

```diff
@@ -0,0 +1,182 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	authclient "github.com/cosmos/cosmos-sdk/x/auth/client"
+	"github.com/cosmos/cosmos-sdk/x/authz/types"
+)
+
+// GetTxCmd returns the transaction commands for this module
+func GetTxCmd() *cobra.Command {
+	AuthorizationTxCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Authorization transactions subcommands",
+		Long:                       "Authorize and revoke access to execute transactions on behalf of your address",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	AuthorizationTxCmd.AddCommand(
+		NewCmdGrantAuthorization(),
+		NewCmdRevokeAuthorization(),
+		NewCmdSendAs(),
```

Comment:
may be `ExecAuthorization` is better here?
```suggestion
		NewCmdExecAuthorization(),
```